From: Kay Sievers <kay@vrfy.org>
Date: Wed, 26 Mar 2014 03:47:03 +0100
Subject: core: do not read system boot timestamps in systemd --user mode

Before:
  $ systemd-analyze --user
  Startup finished in 2.810s (firmware) + 48ms (loader) + 122ms (userspace) = 122ms

After:
  $ systemd-analyze --user
  Startup finished in 122ms (userspace) = 122ms

(cherry picked from commit 530a9662aa3d291555e5b3f6eb43199e1b04f63c)
(cherry picked from commit ba3942b4892023483422c7efdbfd29c58d2ac24b)
---
 src/core/manager.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/core/manager.c b/src/core/manager.c
index 3a38fab..28f4d72 100644
--- a/src/core/manager.c
+++ b/src/core/manager.c
@@ -495,7 +495,7 @@ int manager_new(SystemdRunningAs running_as, bool reexecuting, Manager **_m) {
                 return -ENOMEM;
 
 #ifdef ENABLE_EFI
-        if (detect_container(NULL) <= 0)
+        if (running_as == SYSTEMD_SYSTEM && detect_container(NULL) <= 0)
                 boot_timestamps(&m->userspace_timestamp, &m->firmware_timestamp, &m->loader_timestamp);
 #endif
 
